js/webpack/app.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
       },
       {
         test: /\.js$/,
-        include: /node_modules\/(material-chip-input|ethereumjs-tx)/,
```

Comment:
It shouldn't be. We are still using UglifyJS to pack the bundle, and UglifyJS only understands ES5 syntax. I think the only thing that is ES6 there are `const` and `let` declarations, so we could either kick those out, or change the minifier to Babili or something else.